internal/kokoro/continuous.sh

```diff
@@ -30,6 +30,7 @@ export GOOGLE_APPLICATION_CREDENTIALS=$KOKORO_KEYSTORE_DIR/72523_go_integration_
 # doing when changing / removing the next line.
 
 export GCLOUD_TESTS_GOLANG_PROJECT_ID=dulcet-port-762
+export GCLOUD_TESTS_GOLANG_SECONDARY_BIGTABLE_PROJECT_ID=gcloud-golang-firestore-tests
```

Comment:
Is this correct? Is the bigtable project the same as the firestore project?

Reply:
Bigtable tests run on GCLOUD_TESTS_GOLANG_PROJECT_ID. There is only one test being added as part of this PR that requires a second Bigtable project (GCLOUD_TESTS_GOLANG_SECONDARY_BIGTABLE_PROJECT_ID) i.e. there is very minimal use of second Bigtable project. Setting up a complete new project for a single test does not make sense. So, Firestore project is being reused here.

Reply:
OK